### PR TITLE
Job listing

### PR DESCRIPTION
**Pull request by loayelzobeidy**

### CODE REVIEW

File: nosql/JobListing.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "jobID":"Integer",
```
Comment (mostafa-abdullah) [inline nosql/JobListing.json]: Can you make it `Number` please?

**Comment by loayelzobeidy** [comment]
@mostafa-abdullah  Done

**Comment by mostafa-abdullah** [comment]
Apparently, you accidentally added `companyProfile.json` web service file. Can you remove it because it doesn't belong to this pull request?

**Comment by loayelzobeidy** [comment]
Removed

File: nosql/JobListing.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "jobFunctions":"String",
+  "positionName":"String",
+  "professionLevel":"String",
+  "companyID":"Number",
+  "companyName":"String",
+  "companyLocation":"String",
+  "compnayPicture":"URL/String",
+  "relatedConnections":[{"userID":"Number","firstName":"String","lastName":"String","currentPostion":"String","profilePictureURL":"String"}],
+  "jobBrief":"String",
+  "requiredSkills":["String","String"],
```
Comment (mostafa-abdullah) [inline nosql/JobListing.json]: Can this array have a single element? like `["String"]`, to be compatible with the other schemas

**Pull request merged**